### DISCUSSION TITLE
I deleted my custom domain but still get pointed to it...

### DISCUSSION THREAD

**Discussion by feelvm** [community/community#33275] [Other Feature Feedback, Questions, & Ideas]
I deleted my custom domain from my repository and from verified then reset all DNS records so it's not pointing at Github yet when I try to use Github domain it still points at my custom domain. Even tried to delete and create a new repository with no CNAME file or custom domain added but it still points me to my domain.

What do I do here so my Pages don't get pointed to my custom domain?

Thank you in advance

**Comment by yoannchaudet** [comment]
DNS changes can be slow to propagate everywhere. You seam to have done the two most important steps already:

1. Delete the DNS records from your DNS zone (this is what can be slow to propagate)
2. Remove the custom domain association from your Pages site (I suggest doing that in Pages settings)
